src/components/PreviewOverlay/styles.scss

```diff
@@ -0,0 +1,146 @@
+//Mobile portrait
+.container {
+	position: fixed; 
+	display: flex;
+	height: 100%;
+	width: 100%;
+	padding-bottom: 4em;
+	top: 0;
+	left: 0;
+	z-index: 3;
+	flex-direction: column;
+	align-items: center;
+	justify-content: flex-end;
+	text-align: center;
+	font-size: 1.62rem;
+	background-color: green;
+}
+
+	.details {
+		z-index: 2;
+	}
+
+		.episode {
+			font-size: 1em;
+		}
+
+		.title {
+			font-size: 1.46em;
+		}
+
+		.controls {
+			display: flex;
+			width: 9em;
+			margin-top: 1em;
+			flex-direction: row;
+			justify-content: space-between;
+		}
+
+			.button {
+				display: flex;
+				align-items: center;
+
+				&::before, 
+				&::after {
+					display: inline-block;
+					width: 0.8em;
+					height: 1.1em;
+					background-color: lime;
+				}
+
+				&.prev::before,
+				&.next::after {
+					content: "";
+				}
+
+				&.prev {
+					margin-left: 0;
+					margin-right: auto;
+
+					&::before {
+						margin-right: 1em;
+					}
+				}
+
+				&.next {
+					margin-left: auto;
+					margin-right: 0;
+
+					&::after {
+						margin-left: 1em;
+					}
+				}
+			}
+			
+
+		.close {
+			position: fixed;
+			width: 1.62em;
+			top: 2.62em;
+			left: 50%;
+			font-size: 1em;
+			transform: translateX(-50%);
+		}
+
+			.label {
+				display: none;
+			}
+
+			.icon {
+				position: relative;
+				height: 100%;
+				width: 100%;
+			}
+
+				.bar {						
+					width: 100%;
+					height: 5px;	
```

Comment:
just checking - is `px` what we mean here?

Reply:
yeah it's just copied from the navmenu project. Placeholder for this phase